openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/OpenAPICodeGeneratorTool.java

```diff
@@ -108,8 +108,8 @@ public void execute(ToolContext toolContext) {
             if (openAPI.isEmpty()) {
                 return;
             }
-            // Handle the details with Ballerina.toml option table, if option table is null then by default
-            // client will generate
+            //Extract the details using the tool config options table in Ballerina.toml.
+            //If the options table is not specified in the TOML file, the client will be generated by default.
```

Comment:
```suggestion
            //If the `options` table is not specified in the TOML file, the client will be generated by default.
```
Mention the exact table name.